src/test/java/com/commercetools/helper/formatter/impl/PaypalPlusFormatterImplTest.java

```diff
@@ -0,0 +1,83 @@
+package com.commercetools.helper.formatter.impl;
+
+import org.javamoney.moneta.Money;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(BlockJUnit4ClassRunner.class)
```

Comment:
I'm not sure, but as i know, if you don't have the runner explicitly mentioned on a class - every build tool selects own default one, e.g. the test results could be different on different build tools.